contrib/cfgchanged.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Hook for oxidized to initate validation after a changed config is stored
+# 
+# $OX_EVENT
+# $OX_NODE_NAME
+# $OX_NODE_IP
+# $OX_NODE_FROM
+# $OX_NODE_MSG
+# $OX_NODE_GROUP
+# $OX_NODE_MODEL
+# $OX_JOB_STATUS
+# $OX_JOB_TIME
+# $OX_REPO_COMMITREF
+# $OX_REPO_NAME
+
+Storage="data"
+MyPath=`dirname $0`
+FilePath="$MyPath/$Storage/$OX_NODE_GROUP"
+GitCmd=`which git`
```

Comment:
Pardon my ignorance - it it not possible to call `git` directly here?

Reply:
Yes, but on different distributions or even install method it might end up in different locations.
Also depending upon how scripts are called, different environment variables (PATH) might be available. I see that a lot in cron jobs, so this is pretty fail-safe and universal

Reply:
As I understand it, `which git` would use `$PATH` as well, wouldn't it? So assuming `$PATH` wasn't set (like in cron environments), then this would still fail, right?

Reply:
In this case it most likely would. It's most often in cron tasks the PATH might be an issue, therefore I use which a bit out of habit.